### PR TITLE
Bump PyJWT version from 1.7.1 to 2.3.0

### PR DESCRIPTION
**Pull request by MojixCoder**

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -54,6 +54,15 @@
...
       ]
+    },
+    {
+      "login": "MojixCoder",
+      "name": "Mojix Coder",
+      "avatar_url": "https://avatars.githubusercontent.com/u/76670309?v=4",
+      "profile": "https://github.com/MojixCoder",
+      "contributions": [
+        "maintenance"
+      ]
```
Comment (yezz123) [inline .all-contributorsrc:65]: @MojixCoder could you please delete this cause the bot automatically add it

**Review by yezz123 — APPROVED** [review]

File: .all-contributorsrc (hunk 1 above)
Comment (MojixCoder) [reply .all-contributorsrc]: Done.

**Review by yezz123 — APPROVED** [review]

**Comment by yezz123** [comment]
@MojixCoder Thanks for contributing 🚀

**Comment by yezz123** [comment]
@all-contributors add @MojixCoder for code, bug

**Pull request merged**